v8/payloads/guild.ts

```diff
@@ -204,7 +204,8 @@ export interface APIGuildWidgetMember {
 	username: string;
 	discriminator: string;
 	avatar: string | null;
-	status: string;
+	status: PresenceUpdateStatus;
+	activity: { name: string };
```

Comment:
The `activity` prop isn't present when the member doesn't have a displayed activity.
```suggestion
	activity?: { name: string };
```

Reply:
oh, I had fixed this but forgot to push lol